pyramid_zipkin/request_helper.py

```diff
@@ -71,7 +71,12 @@ def should_not_sample_path(request):
     """
     blacklisted_paths = request.registry.settings.get(
         'zipkin.blacklisted_paths', [])
-    regexes = [re.compile(r) for r in blacklisted_paths]
+    # Only compile strings, since even recompiling existing
+    # compiled regexes takes time.
+    regexes = [
+        re.compile(r) if isinstance(r, str) else r
```

Comment:
lets also handle unicode. and [use basestring](http://stackoverflow.com/a/1979107)